automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -0,0 +1,185 @@
+package org.broadinstitute.dsde.firecloud.api
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.config.{AuthToken, Config}
+
+import scala.concurrent.duration._
+
+
+trait Orchestration extends FireCloudClient with LazyLogging {
+
+  def responseAsList(response: String): List[Map[String, Object]] = {
```

Comment:
Something like:
```
  def parseResponse2[A: ClassTag](response: HttpResponse): A = {
    val f = response.entity.toStrict(5 seconds).map(entity => mapper.readValue(entity.data.toArray, classTag[A].runtimeClass.asInstanceOf[Class[A]]))
    Await.result(f, 100 millis)
  }
```
The `.asInstanceOf` is unfortunate but that's what you get with Jackson (I'm trying to avoid saying use a more idiomatic scala JSON parser :)).